### PR TITLE
Add mission statement and goals

### LINKED ISSUE #68: Add mission statement and statement of goals to website
(as brought up at Nov 2019 PyHC meeting)

### PR DESCRIPTION
**Pull request by sapols**
This PR adds our mission statement and statement of goals to the homepage—between the jumbotron and the News section—and adds an Acknowledgments page—currently populated with just one entry for our graphic designer. It closes #68.

I named this feature branch `shawn-new-pages` because I originally thought the mission statement/goals could go in a separate "About" page, but I didn't end up liking that. The page was weirdly short and felt hidden. That content seemed like something worthy of the home page so I put it there.

I emailed briefly with Dayana to confirm that she likes the text in her Acknowledgment.

### CODE REVIEW

**Comment by sapols** [comment]
Just pushed a commit that makes "Contact" the last nav item again instead of "Acknowledgments". @jibarnum let me know if you have thoughts about that. "Acknowledgments" always come last in science papers, but putting "Contact" last is a pretty standard website convention.

**Comment by jibarnum** [comment]
Overall, I really like the new additions/how you decided to implement them. I wonder if anything else should go in acknowledgements (e.g., others addition to the website overall)? Maybe something to ask the group/in a different PR. The only thing that I would think about changing from what you have now is centering the mission statement text instead of left aligning. It just looks slightly off compared to the jumbotron text, which is centered. But I'm open to having my mind changed about that.

**Comment by sapols** [comment]
@jibarnum I had the exact same thoughts. I definitely want to reach out to the group and see if there are any ideas for other acknowledgments. I feel like someone's gotta have an idea.

As for that text alignment, here's what it looks like centered:
Screen Shot 2020-12-02 at 2 35 56 PM

The thing I didn't like about it is that the mission statement matches what's in the jumbotron but it doesn't match the goals below it (can't center the bulleted list because it makes the bullets not align... ick). I figured the mission statement should match the styling of the goals since the two together are visually separate from the jumbotron.

But I don't hate either option, so if you like it better centered I could be convinced to change it.

**Comment by jibarnum** [comment]
> @jibarnum I had the exact same thoughts. I definitely want to reach out to the group and see if there are any ideas for other acknowledgments. I feel like someone's gotta have an idea.

Sweet, then yeah, I'd send a message to the Element channel. 

> The thing I didn't like about it is that the mission statement matches what's in the jumbotron but it doesn't match the goals below it (can't center the bulleted list because it makes the bullets not align... ick). I figured the mission statement should match the styling of the goals since the two together are visually separate from the jumbotron.
> But I don't hate either option, so if you like it better centered I could be convinced to change it.

I see your point. Looking at the options side by side... It still seems less offensive to my eyes to have the Mission Statement centered. I definitely agree, don't center the bullet points because.... ew. Ha

**Review by jibarnum — APPROVED** [review]

File: _includes/header.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <nav class="navbar navbar-expand-lg navbar-light">
   <!-- Group Logo can go here -->
   <a href="/">
-    <img src="../img/PyHC logo 250x250.png" alt="PyHC Logo" width="52" height="52">
+    <img src="{{ site.baseurl }}/img/PyHC logo 250x250.png" alt="PyHC Logo" width="52" height="52">
```
Comment (sapols) [inline _includes/header.html:4]: Apparently the file structure changes for the secondary blog pages such that `../img/` wasn't a valid directory for them; the logo at the top was appearing as a square with a question mark. Referencing `site.baseurl` instead fixed it.

Didn't notice this until a few minutes ago.

**Pull request merged**